tools/formatter/format.mk

```diff
@@ -28,6 +28,10 @@ format/dart:
 format/line-endings:
 	git ls-files -z | xargs --null ${_start_args} dos2unix --keep-bom --
 
+.PHONY: format/markdown
+format/markdown:
+	markdownlint -c tools/formatter/markdownlint_config.yml '**/*.md' --ignore 'LICENSE.md'
```

Comment:
Is it possible to use `git ls-files -z` output instead of `**/*.md`?
I got an error because I had symlinks in the repo that pointed to some docker image files, and `**` could not read it.
Also `**/*.md` will include any external dependencies that are copied to the repo but added to gitignore. For example, we use it for qti backend. We don't want to format any such files.

Reply:
Ok. Done.